### PR TITLE
[DOC] Frames README: Minor doc fixes and improvements

### PR DESCRIPTION
**Pull request by Sharon-iguazio**
@talIguaz please review and merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -343,11 +343,6 @@ write(backend, table, dfs, condition="", labels=None, max_in_message=0,
     index_cols=None, partition_keys=None)
 ```
 
-- When the value of the [`iterator`](#method-read-param-iterator) parameter is `False` (default) &mdash; returns a single DataFrame.
```
Comment (talIguaz) [inline README.md]: Why did you delete this?

Comment (Sharon-iguazio) [reply README.md]: I deleted the `iterator` info in the `write` section because it's not relevant to `write` and is already documented for `read` (as indicated in the commit message).

**Comment by Sharon-iguazio** [comment]
@talIguaz note that I added another commit (commit 794833b) + see my response to your previous review comment.
